File: flow_sdk/fs_store/storage_layout.py
"""Storage layout types for resource records."""

from enum import Enum


class StorageLayout(str, Enum):
    """How a single record is persisted on disk."""

    FILE = "file"          # standalone <type>-@<uid>.json file
    FOLDER = "folder"      # <type>-@<uid>/ directory with record.json inside

File: ts_sdk/src/resource_management/fs_records/storage-layout.ts
/**
 * StorageLayout — how a record type is persisted on disk.
 *
 * FILE       – one JSON file per record (directory-based storage)
 * LIST_ITEM  – many records inside a single JSON array file
 * FOLDER     – record is a directory containing child files
 */
export enum StorageLayout {
  FILE = 'file',
  LIST_ITEM = 'list_item',
  FOLDER = 'folder',
}
